src/spring/azext_spring/spring_instance.py

```diff
@@ -74,9 +75,12 @@ def create_service(self,
             zone_redundant=zone_redundant
         )
 
-        if enable_log_stream_public_endpoint is not None:
+        if enable_log_stream_public_endpoint is not None or enable_dataplane_public_endpoint is not None:
+            val = enable_log_stream_public_endpoint if enable_log_stream_public_endpoint is not None else \
+                enable_dataplane_public_endpoint
```

Comment:
```suggestion
        if enable_dataplane_public_endpoint is not None:
```
We have already verified the other one is exactly the same in validator. Let us forget that here.